apl-core/src/test/java/com/apollocurrency/aplwallet/apl/data/DexTestData.java

```diff
@@ -17,4 +26,152 @@
     public final DexOrder ORDER_SEA_3 = new DexOrder(1030L, 4L, OrderType.SELL, 100L, DexCurrencies.APL, 400000L, DexCurrencies.ETH, BigDecimal.valueOf(0.001), 8000, OrderStatus.WAITING_APPROVAL, 121, "APL-K78W-Z7LR-TPJY-73HZK", "0x602242c68640e754677b683e20a2740f8f95f7d3");
     public final DexOrder ORDER_BEA_4 = new DexOrder(1040L, 5L, OrderType.BUY, 100L, DexCurrencies.APL, 600000L, DexCurrencies.ETH, BigDecimal.valueOf(0.001), 11000, OrderStatus.OPEN, 122, "0x602242c68640e754677b683e20a2740f8f95f7d3", "APL-K78W-Z7LR-TPJY-73HZK");
+
+    public static final long EXCHANGE_CONTRACT_ID_1       =  -3625894990594689368L ;
+    public static final long EXCHANGE_CONTRACT_ID_2       =  -7277152511870517934L ;
+    public static final long EXCHANGE_CONTRACT_ID_3       =  8455581613897449491L ;
+    public static final long EXCHANGE_CONTRACT_ID_4       =  7952648026362992483L ;
+    public static final long EXCHANGE_CONTRACT_ID_5       =  7952648026362992483L ;
+    public static final long EXCHANGE_CONTRACT_ID_6       =  8455581613897449491L ;
+    public static final long EXCHANGE_CONTRACT_ID_7       =  -6988530272040477515L ;
+    public static final long EXCHANGE_CONTRACT_ID_8       =  4590047955464765433L ;
+    public static final long EXCHANGE_CONTRACT_ID_9       =  -1620743079267768652L ;
+    public static final long EXCHANGE_CONTRACT_ID_10      =  664254800608944568L ;
+    public static final long EXCHANGE_CONTRACT_ID_11      =  -139104235169499924L ;
+    public static final long EXCHANGE_CONTRACT_ID_12      =  -4808040955344135102L ;
+    public static final long EXCHANGE_CONTRACT_ID_13      =  -4084842872841996828L ;
+    public static final long EXCHANGE_CONTRACT_ID_14      =  1035081890238412012L ;
+    public static final long EXCHANGE_CONTRACT_ID_15      =  -2471101731518812718L ;
+
+    public final ExchangeContract EXCHANGE_CONTRACT_1;
+    public final ExchangeContract EXCHANGE_CONTRACT_2;
+    public final ExchangeContract EXCHANGE_CONTRACT_3;
+    public final ExchangeContract EXCHANGE_CONTRACT_4;
+    public final ExchangeContract EXCHANGE_CONTRACT_5;
+    public final ExchangeContract EXCHANGE_CONTRACT_6;
+    public final ExchangeContract EXCHANGE_CONTRACT_7;
+    public final ExchangeContract EXCHANGE_CONTRACT_8;
+    public final ExchangeContract EXCHANGE_CONTRACT_9;
+    public final ExchangeContract EXCHANGE_CONTRACT_10;
+    public final ExchangeContract EXCHANGE_CONTRACT_11;
+    public final ExchangeContract EXCHANGE_CONTRACT_12;
+    public final ExchangeContract EXCHANGE_CONTRACT_13;
+    public final ExchangeContract EXCHANGE_CONTRACT_14;
+    public final ExchangeContract EXCHANGE_CONTRACT_15;
+
+    public final ExchangeContract NEW_EXCHANGE_CONTRACT_16;
+    public final long NEW_EXCHANGE_CONTRACT_ID = -762612439991997299L;
+    public final long NEW_EXCHANGE_CONTRACT_SENDER_ID = -382612439991997299L;
```

Comment:
Maybe better to set here more meaningful ids, such as BOB_ID = 100, or ALICE_ID=200?